src/app/admin/MemberForm.tsx

```diff
@@ -0,0 +1,74 @@
+'use client';
+
+import Autocomplete from '@/components/Autocomplete';
+import { fetcher } from '@/lib/fetcher';
+import { useState } from 'react';
+import useSWRMutation from 'swr/mutation';
+import type { Member } from './page';
```

Comment:
I'm not a huge fan of this cross import situation, even if it does work. Can this `Member` type simply be defined in this file instead?

Reply:
this is not cross import. It's type from it's parent component to keep type-safe.